generated_types/src/job.rs

```diff
@@ -135,6 +140,17 @@ impl From<management::operation_metadata::Job> for Job {
                     chunk_id,
                 },
             },
+            Job::DropPartition(management::DropPartition {
+                db_name,
+                partition_key,
+                table_name,
+            }) => Self::DropPartition {
+                partition: PartitionAddr {
+                    db_name: Arc::from(db_name.as_str()),
```

Comment:
It does not matter at all, but I think you can also make `Arc<str>`s like this:

```suggestion
                    db_name: db_name.as_str().into(),
```

Reply:
I think that `.into` also works, but it's a pattern used within the same file. I can do a cleanup in a follow-up PR.